tests/test_kgain_cal.py

```diff
@@ -122,7 +122,7 @@ def count_contiguous_repeats(arr):
                                divide_em=True)
         image_sim.ext_hdr['DATETIME'] = time_stack_arr0[t+j*exp_repeat_counts[j]]
         # OBSTYPE has no KGAIN value, but NONLIN
-        image_sim.ext_hdr['OBSTYPE'] = 'NONLIN'
+        image_sim.pri_hdr['OBSTYPE'] = 'NONLIN'
```

Comment:
This should be VISTYPE, right?

Reply:
No, this is a unit test. We assume that Sergi's step has already been run and populated OBSTYPE into the header. (the step function still needs OBSTYPE, the DRP now produces it for the step function).